webrtc/RTCPeerConnection-mandatory-getStats.https.html

```diff
@@ -23,18 +23,18 @@
     "packetsReceived",
     "packetsLost",
     "jitter",
-    "packetsDiscarded",
     "framesDropped"
   ],
   RTCInboundRtpStreamStats: [
-    "receiverId",
+    "trackIdentifiedId",
```

Comment:
Typo: should be `trackIdentifier`

Reply:
fixed in https://github.com/web-platform-tests/wpt/pull/35703/commits/8ae7fbe95f35822e022b09fcdb0ef3590ecd9603